mappings/community/convictions/DX12340A-convictions.json

```diff
@@ -629,18 +617,6 @@
             "lengthUnits": "Months",
             "terminationDate": "{{now offset='5 months' format='yyyy-MM-dd'}}",
             "terminationReason": "ICMS Miscellaneous Event",
-            "currentOrderHeaderDetail" : {
-              "custodialTypeCode" : "P",
-              "custodialTypeDescription": "ORA Adult Custody (inc PSS)",
-              "mainOffenceDescription": "Aggravated burglary in a building other than a dwelling (including attempts) - 03100",
-              "status": "Post Sentence Supervision",
-              "disposalDate": "{{now offset='-6 months' format='yyyy-MM-dd'}}",
-              "actualReleaseDate": "{{now format='yyyy-MM-dd'}}",
-              "licenceExpiryDate": "{{now format='yyyy-MM-dd'}}",
-              "pssEndDate": "{{now offset='5 months' format='yyyy-MM-dd'}}",
-              "length": 11,
-              "lengthUnit": "Months"
-            },
```

Comment:
Looking at CCS swagger, I noticed that currentOrderHeaderDetail doesn't return on the probation-record endpoint but does on the single conviction endpoint as part of the sentence, so I have removed from the mock data and updated the pact schema.